### PR TITLE
Added note about unsuported expressions in Python parameters file

### PR DESCRIPTION
**Pull request by daavoo**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Review by shcheklein — APPROVED** [review]

File: content/docs/command-reference/params/index.md [hunk 1]
```diff
@@ -190,6 +190,10 @@ previous version, which is why all `Old` values are `—`.
 
 ## Examples: Python parameters file
 
+> ⚠️ Note that complex expressions (those not supported by
+> [ast.literal_eval](https://docs.python.org/3/library/ast.html#ast.literal_eval))
+> inside the Python parameters file won't be parsed.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/params/index.md:195]:
```suggestion
> ⚠️ Note that complex expressions (unsupported by
> [ast.literal_eval](https://docs.python.org/3/library/ast.html#ast.literal_eval))
> won't be parsed as DVC parameters.
```

File: content/docs/command-reference/params/index.md [hunk 2]
```diff
@@ -204,6 +208,8 @@ SET = {4, 5, 6}
 TUPLE = (10, 100)
 NONE = None
 
+DICT_EXP = dict(a=1, b=2) # Will NOT be found due to the expression
```
Comment (jorgeorpinel) [inline content/docs/command-reference/params/index.md:211]:
```suggestion
# Complex expressions will be ignored.
DICT_EXP = dict(a=1, b=2)
```

File: content/docs/command-reference/params/index.md [hunk 3]
```diff
@@ -204,6 +208,9 @@ SET = {4, 5, 6}
 TUPLE = (10, 100)
 NONE = None
 
+# Complex expressions will be ignored.
+DICT_EXP = dict(a=1, b=2)
```
Comment (jorgeorpinel) [inline content/docs/command-reference/params/index.md:212]: Is that correct? Will they be ignored or will there be some sort of error @daavoo ? Thanks

**Review by jorgeorpinel — APPROVED** [review]
Just one quick doubt but this is probably good to go 🚀

File: content/docs/command-reference/params/index.md (hunk 3 above)
Comment (daavoo) [reply content/docs/command-reference/params/index.md:212]: It is currently ignored. I'm thinking about sending a P.R. to at least log a warning

**Pull request merged**